README.md

```diff
@@ -1,30 +1,288 @@
-Library Name 
-============
+# Easify Exports
 
-Brief description of the component or library.
+The repo contains a series of Easify componenets which are providing the export functionality of any data source to CsV files targeted to 
+  - S3, 
+  - Azure Blob
+  - File Share 
+  - In Memory file store
+  - Or a combination of the targets 
 
-## Get Started
+It also send the notifications on success and error result to an Scheduler (which has been implemented the required interface).
```

Comment:
It also sends the notifications on success and error to a Scheduler (which has implemented the required interface)